src/table_builder.cc

```diff
@@ -146,7 +146,8 @@ void TitanTableBuilder::AddBlob(const ParsedInternalKey& ikey,
 
   // Init blob_builder_ first
   if (!blob_builder_) {
-    status_ = blob_manager_->NewFile(&blob_handle_);
+    // Set the GC's blob file with a high_io pri in ratelimiter
```

Comment:
Seems the comment is wrong. It should be `Flush's`?

Reply:
yes ,you're right.